t/plugin/batch-requests.t

```diff
@@ -716,24 +722,28 @@ passed
                     {
                         "status": 200,
                         "headers": {
-                            "X-Cookie": "request-cookies-b"
+                            "X-Cookie": "request-cookies-b",
+                            "X-HeaderB": "request-header-b"
                         }
                     },
                     {
                         "status": 201,
                         "headers": {
-                            "X-Cookie": "request-cookies-c"
+                            "X-Cookie": "request-cookies-c",
+                            "X-HeaderC": "request-header-c"
                         }
                     },
                     {
                         "status": 202,
                         "headers": {
-                            "X-Cookie": "request-cookies-d"
+                            "X-Cookie": "request-cookies-d",
+                            "X-HeaderD": "request-header-d"
                         }
                     }
                 ]]=],
                 {
-                    Cookie = "request-cookies"
+                    Cookie = "request-cookies",
+                    Header = "request-header"
```

Comment:
that is not a good name, we'd better use a different one.

Reply:
You mean we should not name key like "Cookie" or "Header"?But browner will send cookie into the http header named "Cookie", it is http specification.

Reply:
`Header` is not a good name. `Cookie` is fine.